app/components/pages/SubmissionWizard/steps/Files/ManuscriptUpload.js

```diff
@@ -123,6 +108,21 @@ const DropzoneContent = ({
       </React.Fragment>
     )
   }
+  if (conversion.converting) {
```

Comment:
While the reasoning behind moving this down seems straight forward the component isn't clearing down the `errorMessage` when a new valid file is being uploaded so while the replacement file is being uplaoded you are still shown the error message with this change.